sdk_contrib/aws_v1_support/src/main/java/io/opentelemetry/sdk/contrib/trace/aws/Ec2Resource.java

```diff
@@ -55,6 +55,7 @@ static Resource getResourceFromInfoAndHost(
       labels.put(ResourceConstants.HOST_ID, info.getInstanceId());
       labels.put(ResourceConstants.HOST_NAME, info.getPrivateIp());
       labels.put(ResourceConstants.HOST_TYPE, info.getInstanceType());
+      labels.put(ResourceConstants.HOST_IMAGE_ID, info.getImageId());
```

Comment:
Some tests?

Reply:
I just realized I missed that and have pushed the change